ci(orc8r): Connect Docusaurus to SwaggerHub API (#8231)
magma/magma: docs/readmes/orc8r/dev_rest_api.md

---
id: dev_rest_api

> title: Northbound REST API

Reviewer: ```suggestion
title: REST API
```

hide_title: true
---

# Northbound REST API

> Orc8r provides a set of northbound REST API that exposes configurations and metrics for network devices. For a quick preview of the API, visit [our most up-to-date API settings on SwaggerHub](https://app.swaggerhub.com/apis/MagmaCore/Magma/1.0.0) (no login required).

Reviewer: ```suggestion
Orc8r provides a northbound REST API that exposes configurations and metrics for network devices. For a quick preview of the API, check out [`MagmaCore/Magma` on SwaggerHub](https://app.swaggerhub.com/apis/MagmaCore/Magma/1.0.0).
```
